src/main/java/io/quarkus/gizmo2/creator/BlockCreator.java

```diff
@@ -2550,6 +2582,20 @@ default void throw_(Class<? extends Throwable> type, String message) {
         }
     }
 
+    /**
+     * Throw a new exception of the given type with a message.
+     *
+     * @param type the exception type
```

Comment:
Same here; `type` is not nullable but `message` is nullable.

Reply:
Done.